etc/profile-m-z/torbrowser.profile

```diff
@@ -0,0 +1,22 @@
+# Persistent local customizations
+include torbrowser.local
+# Persistent global definitions
+include globals.local
+
+ignore dbus-user none
+
+noblacklist ${HOME}/.cache/mozilla
+noblacklist ${HOME}/.mozilla
+
+blacklist /usr/libexec
+
+mkdir ${HOME}/.cache/mozilla/torbrowser
+mkdir ${HOME}/.mozilla
+whitelist ${HOME}/.cache/mozilla/torbrowser
+whitelist ${HOME}/.mozilla
+include whitelist-usr-share-common.inc
+
+dbus-user filter
+dbus-user.own org.mozilla.torbrowser.*
+
+include firefox-common.profile
```

Comment:
```suggestion
# Firejail profile for torbrowser
# This file is overwritten after every install/update
# Persistent local customizations
include torbrowser.local
# Persistent global definitions
# added by included profile
#include globals.local

# Redirect
include start-tor-browser.profile
```

Reply:
www-client/torbrowser::torbrowser on Gentoo Linux doesn't use torbrowser-launcher. It compiles torbrowser directly from source code.

Reply:
Ahh, it is only for the gentoo package, then add this information to the description.

Reply:
I just did in my local copy. It will be reflected soon in the pull request.